tests/pyincore_data/test_pyincore_data.py

```diff
@@ -31,3 +31,20 @@ def test_get_fips_by_state_county():
     fips = CensusUtil.get_fips_by_state_county("illinois", "champaign")
 
     assert fips == '17019'
+
+
+def test_national_ave_values():
+    navs = CensusUtil.national_ave_values(year=2020)
+    assert navs[1]['average'] == 0.644373548235299
+
+def test_demographic_factors():
+    state = "texas"
+    county = "galveston"
+    year = 2020
+    fips = CensusUtil.get_fips_by_state_county(state=state, county=county)
+    state_code = fips[:2]
+    county_code = fips[2:]
+    geo_type = "block%20group:*"
+    social_vunlnerability_dem_factors_df = CensusUtil.demographic_factors(state_code=state_code, county_code=county_code,
```

Comment:
the line has pep8 warning because it is little too long

Reply:
would you be able to run \pyincore-data\tests\test_format.py 
it will let you know where to fix in terms of pep8 warnings. There are quite a lot of places to fix.

Reply:
Oh my bad, I will fix them right away!